Makefile

```diff
@@ -77,50 +77,9 @@ watch-unit:
 watch-compile:
 	$(BIN_DIR)/tsc --watch  --noEmit
 
-e2e-in-ci:
-	make create-tmp-env-ci && \
-	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml up e2e-tests
-
-reset-e2e-in-ci-with-build: reset-deps e2e-in-ci-with-build
-
-e2e-in-ci-with-build:
-	yarn build && \
-	make create-tmp-env-ci && \
-	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml run --name e2e-tests e2e-tests make execute-e2e-from-within-container-cached || \
-	docker rm `docker ps -q -f status=exited`
-
-e2e-in-ci-cached:
-	make create-tmp-env-ci && \
-	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml run --name e2e-tests e2e-tests make execute-e2e-from-within-container-cached || \
-	docker rm `docker ps -q -f status=exited`
-
-delete-e2e:
-	docker container kill `docker ps -f name="e2e-tests" -q` && \
-	sleep 1 && \
-	docker container rm `docker ps -f name="e2e-tests" -q`
-
-main-in-ci-build:
-	yarn build && \
-	make create-tmp-env-ci && \
-	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml -f docker-compose.override.yml run --name e2e-tests e2e-tests make start-main-ci || \
-	docker rm e2e-tests
-
-main-in-ci-cached:
-	make create-tmp-env-ci && \
-	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml run --name e2e-tests e2e-tests make start-main-ci || \
-	docker rm e2e-tests
-
 del-containers:
 	docker compose rm -sfv
 
-execute-e2e-from-within-container:
```

Comment:
there is a reference in quickstart and docker compose

Reply:
reference has been removed